<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout1">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gracefulShutdownButton">
       <property name="text">
        <string>Graceful Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleQuitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gracefulShutdownButton</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>handleGracefulQuitButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleQuitButton()</slot>
  <slot>handleGracefulQuitButton()</slot>
 </slots>
</ui>
